### PR TITLE
glab: add v1.19.0

### PR DESCRIPTION
**Pull request by profclems**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/24335)

### CODE REVIEW

File: manifests/g/GLab/GLab/1.19.0/GLab.GLab.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+PackageIdentifier: GLab.GLab
+PackageName: GLab
```
Comment (ItzLevvie) [inline manifests/g/GLab/GLab/1.19.0/GLab.GLab.yaml:2]:
```suggestion
PackageName: glab
```

File: manifests/g/GLab/GLab/1.19.0/GLab.GLab.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- utility
+- tool
+InstallerType: inno
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/profclems/glab/releases/download/v1.19.0/glab_1.19.0_Windows_x86_64_installer.exe
+  InstallerSha256: 9523631AF4D3B2FF756E4B085C829CAFBE2FE9F6F4B4AE5995C46A622FAC4D28
+  InstallerSwitches:
+    Silent: /VERYSILENT /ALLUSERS
+    SilentWithProgress: /SILENT /ALLUSERS
```
Comment (ItzLevvie) [inline manifests/g/GLab/GLab/1.19.0/GLab.GLab.yaml:28]:
```suggestion
```
I believe these aren't needed as winget-cli automatically runs /SILENT on inno installers?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

File: manifests/g/GLab/GLab/1.19.0/GLab.GLab.yaml (hunk 2 above)
Comment (OfficialEsco) [reply manifests/g/GLab/GLab/1.19.0/GLab.GLab.yaml:28]: As far as i could see the installer didn't require /ALLUSERS either

**Review by OfficialEsco — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**